packages/@aws-cdk/aws-amplify/README.md

```diff
@@ -15,8 +15,79 @@
 ---
 <!--END STABILITY BANNER-->
 
```

Comment:
Provide some background about what is the Amplify service and what is this library capable of doing. As much as possible copy details and links from AWS documentation.

Reply:
The Amplify service (known as the Amplify Console) offers continuous deployment and fully managed hosting for fullstack serverless web apps.